web_monitoring/html_diff_render.py

```diff
@@ -284,6 +287,9 @@ def html_diff_render(a_text, b_text, a_headers=None, b_headers=None,
                 'meta',
                 content=_diff_title(soup_old, soup_new))
             title_meta.attrs['name'] = 'wm-diff-title'
+            if not soup.head:
+                head = soup.new_tag('head')
+                soup.insert(0, head)
```

Comment:
This doesn’t work right. I’m guessing you added it to fix the tests, but it could significantly mangle the output. I took a look and it turns out there’s a bug in html5-parser’s `soup` tree builder: https://github.com/kovidgoyal/html5-parser/issues/18

We can add a workaround for now or we can wait for it to be fixed, but I don’t think we can merge this as-is.

Reply:
Oh you fixed the `html5-parser` bug, great! I'll remove my workaround.

Reply:
Hopefully it gets released soon!

Reply:
@Mr0grog still no release for `html-parser` :(

Reply:
0.4.7 is out now!

Reply:
After using `html5-parser 0.4.7` and removing the workaround I see that the tests are fixed.